mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DefaultStatementAttributes.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.mapper;
+
+import com.datastax.oss.driver.api.core.ConsistencyLevel;
+import com.datastax.oss.driver.api.core.config.DriverExecutionProfile;
+import com.datastax.oss.driver.api.core.metadata.Node;
+import com.datastax.oss.driver.api.mapper.StatementAttributes;
+import com.datastax.oss.driver.api.mapper.StatementAttributesBuilder;
+import com.datastax.oss.protocol.internal.util.collection.NullAllowingImmutableMap;
+import java.nio.ByteBuffer;
+import java.time.Duration;
+import java.util.Map;
+
+public class DefaultStatementAttributes implements StatementAttributes {
+
+  private DriverExecutionProfile profile = null;
+
+  private String profileName = null;
+
+  private Boolean isIdempotent = null;
+
+  private int pageSize = -Integer.MIN_VALUE;
+
+  private ConsistencyLevel consitencyLevel = null;
+
+  private ConsistencyLevel serialConsistencyLevel = null;
+
+  private Duration timeout = null;
+
+  private ByteBuffer routingKey = null;
+
+  private String routingKeyspace = null;
+
+  private boolean tracing = false;
+
+  private ByteBuffer pagingState = null;
+
+  private long timeStamp = Long.MIN_VALUE;
```

Comment:
```suggestion
  private long timestamp = Long.MIN_VALUE;
```
For consistency with the getter name (and this is the capitalization we've been using so far in the driver).